### ISSUE TITLE
php config docs missing

### ISSUE THREAD

**Issue by glensc** [theorchard/monolog-cascade#48]
The [readme](https://github.com/theorchard/monolog-cascade#configuring-your-loggers) says it supports YAML, JSON and PHP configs, but i can't find any example or documentation how the file should be formatted, should it return array and in what structure?

I end up instead with `Illegal string offset` error, similar to #41

**Comment by glensc** [comment]
reproducer: https://gist.github.com/glensc/d2a265723abb31df4257

gives now:
```
PHP Fatal error:  Uncaught exception 'Symfony\Component\Config\Exception\FileLoaderLoadException' with message 'Cannot load resource "/tmp/issue-48/config.php".' in /tmp/issue-48/vendor/symfony/config/Loader/DelegatingLoader.php:42
```

**Comment by glensc** [comment]
thanks @michaelmoussa, exactly what i'm looking for!

**Comment by rantonmattei** [comment]
I wish I had more time to polish this part cause it is a bit confusing indeed. the `fileConfig` function name is misleading. It may take an array.

```php
$option = array(
    // you config array here
);
Cascade::fileConfig($options);
```

@michaelmoussa 's PR is a good alternative.

**Comment by glensc** [comment]
can you merge it? i then can base my changes on his code.

i can send you PR based on your https://github.com/theorchard/monolog-cascade/issues/48#issuecomment-185257606

**Comment by rantonmattei** [comment]
I merged #50
